src/.storybook/main.js

```diff
@@ -1,4 +1,13 @@
 const path = require('path');
+const outlineConfig = require('../../outline.config');
+
+const excludedStories = outlineConfig.excludedStories;
+function getExcluded() {
+  if ((excludedStories.length = 1)) {
```

Comment:
What does this do? When would this fail?

Reply:
This prevents the | from being added if you're only excluding one story.